cpp/include/cuspatial/trajectory.hpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/cudf.h>
+
+namespace cuspatial {
+
+/**
+ * @Brief deriving trajectories from points (x/y relative to an origin), timestamps and objectids
+ * by first sorting based on id and timestamp and then group by id.
```

Comment:
Not part of the @brief.
```suggestion
 *
 * Sorts by object id and timestamp and then groups by id.
```